torch/jit/_recursive.py

```diff
@@ -394,6 +394,16 @@ def init_fn(script_module):
         if _jit_internal.get_torchscript_modifier(item) is _jit_internal.FunctionModifiers.COPY_TO_SCRIPT_WRAPPER:
             add_python_attr_to_scripted_model(script_module, nn_module, name)
 
+    # Set up hooks for eager mode
```

Comment:
This copies them only at the Python level, I don't know if we want to have this since it means some things are running inside the TorchScript interpreter (the model's `forward`) and some things are running outside in Python (the hooks), which seems like it could be confusing

Reply:
We can drop this. I thought this could be useful since we already support the following code pattern although the hooks don't get executed. I was just making it more inline with the eager mode.
```
# class M(nn.Module)
m = M()
scripted_m = torch.jit.script(m)
scripted_m(torch.tensor(0))
```